app/services/tts_generator.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "google/cloud/text_to_speech"
+require "yaml"
+require "stringio"
+
+## TODO: for queue job
+# return unless word.with_tts?
+# word.audio.attach(
+#       io: TtsGenerator.call(word.name),
+#       filename: 'audio.mp3',
+#       content_type: 'audio/mp3'
+#     )
+
+# Generates a TTS mp3 file for a given string via Google Cloud Text to Speech service and returns
+# an StringIO object. Configuration in config/tts.yml
+#
+# Usage:
+#   TtsGenerator.call('Hallo Welt')
+class TtsGenerator
```

Comment:
I generally try to avoid abbreviations, as I prefer readability over brevity. `TextToSpeechGenerator` might be more readable. No need to change now, but maybe for next time :slightly_smiling_face: